### DISCUSSION TITLE
Strategic Item Banning Mechanism in Gaming: Enhancing Engagement and Tactical Diversity

### DISCUSSION THREAD

**Discussion by 3nines** [ValveSoftware/Dota2-Gameplay#15178] [Ideas]
After being automatically banned for 24 hours by the system due to my uninterested teammates because i said BUU!, I finally got the time to present an idea that I've been pondering for a long time.

If you strongly dislike a specific item and want to see it limited for a while, this idea might be particularly suitable, without putting extra pressure on you or your opponents.

For several years, I've considered the possibility of banning items in the game, but I've neither had the time nor the energy to propose it. Having stopped posting suggestions on Reddit due to negative behaviors there, I now choose to present my idea here, hoping we can gather for a discussion about whether this could benefit all of us in the game.

**So how does it work? According to the proposal:**
![order](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/2ca5d730-4426-47e4-8935-6a4649baa41a)

Each team gets a list on the screen with 12 items to ban for 20 sec, based on the heroes they have chosen. Each player in the team can vote for three of these items. The item with the most votes gets banned for 44 minutes, the next most voted for 34 minutes, and the one with the fewest votes for 21 minutes.

**Bear in mind that the same items may be banned for both teams, particularly if they are commonly used with their selected heroes, but banned for different time.**

The time table is by fibonacci sequence number [https://en.wikipedia.org/wiki/Fibonacci_sequence](url)

By order: 
Item ban for: 21:00 min
Item ban for: 34:00 min
Item ban for: 44:00 min

![12_items](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/27f0224d-1118-41c4-9763-a7903f8618c2)

Ingame restock time:

![restock_time](https://github.com/ValveSoftware/Dota2-Gameplay/assets/3405316/0943dea7-2e24-4916-afa5-2d629095861e)

I strongly believe that banning certain items for a period could make the game more attractive and enjoyable. For instance, when a new patch is released, it often introduces new items or changes to existing ones, leading to their frequent use in pub rank.

What do you think, dear friends? Should we implement this proposal to enhance our enjoyment of the game? If it were up to me, this feature would have been a part of the game long ago.

What does the dev dota 2 team think of this idea? :)

**Comment by commentator24** [comment]
Banning not a good idea because it gives unfair advantages. e.g. ban diffusal when playing as medusa. ban pipe with strong magic team. ban halberds with strong right click heroes. ban blink when enemies wouldve needed it to be useful e.g. axe, sandking, earth shaker. or even ban aghs / shards. 

But I would've love to see Stacking effects at higher prices / time cooldown though. e.g. 2 desolator. first desolator is cheap. to buy second one will need 1500 g recipe etc. 
e.g. team accidentally buy 2 cuirass / guardian greave / radiance. maybe 2nd greave should have 1500g more expensive recipe? or has 7 min cooldown to buy another recipe for team. This way team is aware someone already has it.

i think it would also be interesting if some items become cheaper/expensive/ sharable recipe if team already has it. e.g. First midas maybe 1750g recipe. next person buy midas recipe is 1500g, then 1250g, 1000g etc. If they buy then resell, obviously the recipe will return to original price. Then we may see team planning for similar items e.g. 5 midas, 5 dagon. 5 blink. 5 aghs etc which make the game outcome more unpredictable. e.g. if a solo player only has midas at 20 min when early game is losing, it makes the game harder because of this player's decision. but if team can take advantage to buy more midas for themselves at cheaper price, this could be useful and cause the responsibility to fall on the team decision as well if not taking advantage of it. e.g. maybe someone richer couldve get midas first so the next person can buy it faster instead of finishing it at 20 min.

**Comment by 3nines** [comment]
`Banning not a good idea because it gives unfair advantages. e.g. ban diffusal when playing as medusa. ban pipe with strong magic team. ban halberds with strong right click heroes. ban blink when enemies wouldve needed it to be useful e.g. axe, sandking, earth shaker. or even ban aghs / shards.`

The essence of the concept is to limit the availability of certain items. It seems that the details of my explanation were not entirely clear. Temporarily limiting certain items is a feature that already exists in the game. For instance, 'shards' can be purchased after 15 minutes.

However, a new opportunity is now offered: to limit access to three popular items for the opposing team, and the same applies to your own team. This forces players to choose alternative items, which in turn makes the game more engaging and entertaining.

Take, for example, Enigma, who cannot purchase a 'dagger' until after 21 minutes. However, Enigma can still buy and use a 'force staff', and Enigma's team can restrict access to BKB for the opposing team.

You also mentioned Earthshaker without a 'blink dagger'. But it's worth noting that his upgraded Enchant Totem to 'aghanim's scepter' provides him with a form of automatic 'blink' in the air.

Introducing restrictions on certain popular items during a match for both teams offers several advantages. It broadens the game's dynamics, requires more strategic thinking, and promotes teamwork.

**Comment by sloperun3az** [comment]
Hi! After reading a few of the postings, I realized that I had never been to this website before, even though I could have sworn that I had been there before. On the other hand, I can't express how happy I am that I discovered <a href="https://cookie-clicker.io/">**cookie clicker**</a> in and make sure to check back!
